test/mysql_flavor.py

```diff
@@ -139,25 +147,28 @@ def change_master_commands(self, host, port, pos):
         (host, port)]
 
 
-__mysql_flavor = None
+MYSQL_FLAVOR = None
 
 
 # mysql_flavor is a function because we need something to import before the
-# actual __mysql_flavor is initialized, since that doesn't happen until after
+# variable MYSQL_FLAVOR is initialized, since that doesn't happen until after
 # the command-line options are parsed. If we make mysql_flavor a variable and
 # import it before it's initialized, the module that imported it won't get the
 # updated value when it's later initialized.
 def mysql_flavor():
-  return __mysql_flavor
+  return MYSQL_FLAVOR
 
 
 def set_mysql_flavor(flavor):
   """Set the object that will be returned by mysql_flavor().
 
   If flavor is not specified, set it based on MYSQL_FLAVOR environment variable.
+
+  Args:
+    flavor: String of the MySQL flavor e.g. "MariaDB" or "MySQL56".
   """
 
-  global __mysql_flavor
+  global MYSQL_FLAVOR
```

Comment:
The linter thinks it's a constant and therefore it wants it all caps. (Constants must be written all caps: https://www.python.org/dev/peps/pep-0008/#constants)